utils/Command.js

```diff
@@ -15,6 +17,9 @@ class Command {
     }
 
     process(message, args) {
+        if (this.adminOnly && !message.member.roles.get(STAFF_ROLE))
```

Comment:
`if (this.adminOnly && !(message.guild && message.member.roles.has(STAFF_ROLE)))`